Source/WebKit/NetworkProcess/storage/OriginQuotaManager.h

```diff
@@ -39,7 +39,7 @@ class OriginQuotaManager : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPt
     using IncreaseQuotaFunction = Function<void(QuotaIncreaseRequestIdentifier, uint64_t currentQuota, uint64_t currentUsage, uint64_t requestedIncrease)>;
     using NotifySpaceGrantedFunction = Function<void(uint64_t)>;
     static Ref<OriginQuotaManager> create(uint64_t quota, uint64_t standardReportedQuota, GetUsageFunction&&, IncreaseQuotaFunction&& = { }, NotifySpaceGrantedFunction&& = { });
-    uint64_t reportedQuota() const;
+    uint64_t reportedQuota();
```

Comment:
I would tend to go with opposite way and make usage() const, with a const_cast for lazy init.

Reply:
Since we are going to keep increasing `m_standardReportedQuota`, this cannot be marked const.